src/solver/simulation/sim_calcul_economique.cpp

```diff
@@ -535,13 +535,15 @@ void SIM_RenseignementProblemeHebdo(PROBLEME_HEBDO& problem,
             for (uint k = 0; k != linkCount; ++k)
             {
                 auto& lnk = *(studyruntime.areaLink[k]);
-                /* gp : these asserts will be done in another way later
-                assert((uint)indx < lnk.data.height);
-                assert((uint)fhlNTCDirect < lnk.data.width);
-                assert((uint)fhlNTCIndirect < lnk.data.width);
-                */
-                ntc->ValeurDeNTCOrigineVersExtremite[k] = lnk.directCapacities[0][indx];
-                ntc->ValeurDeNTCExtremiteVersOrigine[k] = lnk.indirectCapacities[0][indx];
+                const int tsIndex
+                  = NumeroChroniquesTireesParInterconnexion[numSpace][k].TransmissionCapacities;
```

Comment:
See remark in **sim_calcul_adequation.cpp**

Reply:
As discussed, I think `numSpace` is right here.